Documentation/network/concepts/ipam/cluster-pool.rst

```diff
@@ -45,6 +45,16 @@ Configuration
 For a practical tutorial on how to enable this mode in Cilium, see
 :ref:`gsg_ipam_crd_cluster_pool`.
 
+Expanding the cluster pool
+==========================
+
+Don't change any existing elements of the ``clusterPoolIPv4PodCIDRList`` list, as
+changes cause unexpected behavior. If the pool is exhausted,
+add a new element to the list. The minimum mask length is ``/30``, with a recommended minimum mask 
```

Comment:
One quick wording nit
```suggestion
add a new element to the list instead. The minimum mask length is ``/30``, with a recommended minimum mask 
```

Reply:
@darox 👋🏻 Please address this comment and re-request review.

Reply:
@zacharysarah I see this change in the PR now, do you see it?

Reply:
@joestringer I do now, thanks for the prompt.